src/cmds/init/initUtils.ts

```diff
@@ -138,19 +163,22 @@ export async function buildWeb3Wallet(argv: YargsArgs): Promise<[
 
   return endWeb3Wallet();
 
-  function endWeb3Wallet(): ([
+  function endWeb3Wallet(): [
     ManifestWalletProperty,
     {
       port: number;
       dist: string;
       outfileName: string;
     },
-  ]) {
+  ] {
```

Comment:
not related to this PR, but having this type inline is confusing to read with the formatting

Reply:
If we lave this comment unresolved I'll get back to it!